comit/src/network/protocols/orderbook.rs

```diff
@@ -179,19 +179,9 @@ impl Orderbook {
     }
 }
 
-#[derive(thiserror::Error, Debug, Clone, Copy)]
-pub enum OrderbookError {
-    #[error("could not make order")]
-    Make,
-    #[error("could not take order because identities not found")]
-    IdentitiesForOrderNotFound(OrderId),
-    #[error("could not take order because not found")]
-    OrderNotFound(OrderId),
-    #[error("could not subscribe to all peers for topic")]
-    Subscribe,
-    #[error("could not unsubscribe to all peers for topic")]
-    UnSubscribe,
-}
+#[derive(PartialEq, Clone, Copy, Debug, thiserror::Error)]
+#[error("order not found in orderbook: {0:?}")]
```

Comment:
```suggestion
#[error("order {0} not found in orderbook")]
```

1. `Display` instead of `Debug` makes for a nicer output
2. Putting the ID in the sentence is closer to an English sentence

Thoughts?

Reply:
Typically I prefer to have the bit of an error message that changes at the end so its easier to visually parse the logs. That might just be a personal thing though. It makes it easier to write log statements that are uniform since so one can scan down the logs and assume any time a id appears it will be at the end. Although now I write this I realise that with line wrapping maybe this is not a real benefit? I'm not fussed which way we go but I'd like to be uniform.

Reply:
Display vs Debug, you are definitely correct. thanks

Reply:
We are talking about errors here and not general log statements though. For logs, I'd say the bit that changes should ideally be in a tracing context otherwise you are likely going to repeat it over several statements.

Since errors cross function boundaries, they are usually logged outside of the scope they were created in and then it helps if they are a self-contained and easily readable message.